### ISSUE TITLE
 Unexpected type '' encountered while reading 'action manifest root'

### ISSUE THREAD

**Issue by wolfram77** [paambaati/codeclimate-action#577]
I am getting the following error when i try to codeclimate-action@v3.

```bash
Error: paambaati/codeclimate-action/v3/action.yml:
Error: paambaati/codeclimate-action/v3/action.yml: (Line: 30, Col: 51, Idx: 902) - (Line: 30, Col: 80, Idx: 931): While parsing a block mapping, did not find expected key.
Error: System.ArgumentException: Unexpected type '' encountered while reading 'action manifest root'. The type 'MappingToken' was expected.
   at GitHub.DistributedTask.ObjectTemplating.Tokens.TemplateTokenExtensions.AssertMapping(TemplateToken value, String objectDescription)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Fail to load paambaati/codeclimate-action/v3/action.yml
```

I think this is being caused by the quotes in `verifyDownload` description in the `actions.yml` file.

Here is the action i am using (simplified):

```yml
on: [push]
jobs:
  coverage:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v2
        with:
          node-version: 18.x
      - run: npm ci
      - run: npm test
      - uses: paambaati/codeclimate-action@v3
        env:
          CC_TEST_REPORTER_ID: <id>
```

**Comment by paambaati** [comment]
See https://github.com/paambaati/codeclimate-action/pull/557#issuecomment-1155390601

**Comment by wolfram77** [comment]
Yes i see the issue in v3 branch, not in v3.0.0. It works now. Thank you.

**Comment by yukihiko-shinoda** [comment]
It's better to take some actions for users. Ex:

- Remove v3 tag at least temporary
- Note this Issues in README.md

When GitHub Actions user check the page: [Tags · paambaati/codeclimate-action](https://github.com/paambaati/codeclimate-action/tags), user will judge that this GitHub Actions supports pinning major versions.

cf. [About custom actions - GitHub Docs](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions)
> you can recommend that your users specify a major version when using your action, and only direct them to a more specific version if they encounter issues.

Users who get lost in the maze waste time to investigate such a complex issue: https://github.com/paambaati/codeclimate-action/pull/557#issuecomment-1155390601

**Comment by paambaati** [comment]
> Note this Issues in README.md

This has now been addressed in 34d2b9c.
